src/styling/themes.ts

```diff
@@ -7,6 +7,7 @@ export const lightTheme = {
 	colors: {
 		...DefaultTheme.colors,
 		borderGrey: '#0000002E',
+		borderGreen: '#325637',
```

Comment:
I suggest we ask Ivanna if its possible for border color to be the same as primary color. I don't see the point in having 50 shades of green in project